plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -1464,24 +1744,6 @@ func TestAssignSecurityContext(t *testing.T) {
 			t.Errorf("%s expected validation errors but received none", k)
 			continue
 		}
-
-		// if we shouldn't have validated ensure that uid is not set on the containers
```

Comment:
removed since we pass copies of the pod to `assignSecurityContext`, rather than mutating and rolling back on failure